### ISSUE TITLE
Bug with 24.4.0: `--crop` not working when invoking from CLI

### ISSUE THREAD

**Issue by leuchtum** [jgraph/drawio-desktop#1728]
### Preflight Checklist

* [x] I agree to follow the [Code of Conduct](https://github.com/jgraph/drawio-desktop/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for a feature request that matches the one I want to file, without success.

You must agree to search and the code of conduct. You must fill in this entire template. If you delete part/all or miss parts out your issue will be closed.

**Describe the bug**
When invoking drawio from the CLI, the `--crop` option does not show any effect. 
I've checked versions 20.7.5 and 24.2.5, there the `--crop` option worked as expected.

**To Reproduce**
1. Make sure you are on 24.4.0
1. Create dummy file with some small content and a large paper size, e.g. A4
2. Run `/Applications/draw.io.app/Contents/MacOS/draw.io -x --crop dummy.drawio`
3. Open output

**Expected behavior**
1. Make sure you are NOT on 24.4.0
1. Create dummy file with some small content and a large paper size, e.g. A4
2. Run `/Applications/draw.io.app/Contents/MacOS/draw.io -x --crop dummy.drawio`
3. Open output

**Screenshots**
Dummy **DRAWIO** file
![Bildschirmfoto 2024-05-16 um 23 21 53](https://github.com/jgraph/drawio-desktop/assets/59877932/45106307-33d1-47f6-8336-8f829eb2d100)

Expected **PDF** file
![Bildschirmfoto 2024-05-16 um 23 23 37](https://github.com/jgraph/drawio-desktop/assets/59877932/2830a1c4-402b-4641-8c30-6d93b8f6218b)

Bug **PDF** file
![Bildschirmfoto 2024-05-16 um 23 24 45](https://github.com/jgraph/drawio-desktop/assets/59877932/1aad73df-a5ac-419f-93ef-8e9fd221c3bd)

**draw.io version (In the Help->About menu of the draw.io editor):**
Ok version: != 24.4.0
Bug version: == 24.4.0

**Desktop (please complete the following information):**
```shell
> sw_vers
ProductName:		macOS
ProductVersion:		14.4.1
BuildVersion:		23E224
```

**Additional context**
NA

**Comment by AlbertHansen** [comment]
This is also a problem using the '''drawio-desktop-bin''' from the AUR, with the version:
```
>>drawio --version
Checking for beta autoupdate feature for deb/rpm distributions
Found package-type: deb
24.4.0
```

**Comment by leuchtum** [comment]
Not sure if this issue should be moved to the https://github.com/jgraph/drawio repo? Since this issue is not desktop related.

**Comment by davidjgraph** [comment]
It is desktop only, only desktop has a CLI.

**Comment by davidjgraph** [comment]
Fix in 24.4.6.
